File: Desenvolvimento/Wear/wear/src/main/java/com/example/soscardio/MainActivity.java
package com.example.soscardio;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import com.example.soscardio.databinding.ActivityMainBinding;

public class MainActivity extends Activity {

    private TextView mTextView;
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        mTextView = binding.text;
    }
}

File: Desenvolvimento/Mobile-Wear/Arquivos de Desenvolvimento - Android Studio/mobile/src/main/java/com/example/soscardio/MainActivity.kt
package com.example.soscardio

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.webkit.WebView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val myWebView: WebView = findViewById(R.id.webviewlogin)
        myWebView.loadUrl("http://10.0.0.181/index.php")

    }
}